utils/statistics-flow-rejection-reason-results.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var db = require('../db');
+
+module.exports = function (reasons) {
+	return reasons.map(function (reason) {
+		var result = [], reasonsConcat = [];
+		reason.rejectionReasons.forEach(function (reasonItem) {
```

Comment:
I think that what was also expected was to include some kind information whether given reason was about data or document: https://unctad.atlassian.net/browse/EV2CORE-3

`Document 1 - the document is not readable", "Document N - the document does not correspond to the expected document", etc. "Data - lorem ipsum lorem ipsum, etc."`

Let's add Document N - and Data - suffixes, and nothing for the main reason if we have one.